src/service/pluginService.ts

```diff
@@ -9,8 +9,8 @@ import * as util from "../util";
 const apiPath =
   "https://marketplace.visualstudio.com/_apis/public/gallery/extensionquery";
 
-const extensionDir: string = ".vscode";
-const extensionDirPortable: string = "/data/extensions/";
+const extensionDir: string = ".vscode/";

```

Comment:
adding `/` caused in this string stopped the syncing

Reply:
Exactly! It's because the path delimiter.